atom/browser/notifications/linux/libnotify_loader.cc

```diff
@@ -1,8 +1,8 @@
-// This is generated file. Do not modify directly.
-// Path to the code generator:
-// tools/generate_library_loader/generate_library_loader.py .
+// Copyright (c) 2015 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
 
-#include "brightray/browser/linux/libnotify_loader.h"
```

Comment:
With GN there is `//tools/generate_library_loader/generate_library_loader.gni` to generate these files during build, can you add a target for generating these. An example config https://cs.chromium.org/chromium/src/third_party/speech-dispatcher/BUILD.gn.

Reply:
Can you maybe help with that? I don't have the Linux build configured yet. Or I could do it later in a follow up PR.

Reply:
Lets fix this in a follow up, not a blocker for this PR. It would be best if you try it, or let me know, I can add it. Just leave a `FIXME` comment in these files.